packages/lesswrong/components/ultraFeed/UltraFeedItemFooter.tsx

```diff
@@ -229,6 +233,8 @@ const UltraFeedItemFooterCore = ({
     </div>
   );
 
+  const votingSystem = voteProps.document.votingSystem || getDefaultVotingSystem();
```

Comment:
votingSystem is is defined when this is used in the bookmarks page, but in other places it may not be. cc @darkruby501 @b0b3rt you may want to fix this more thoroughly

Reply:
I'm kind of surprised there was an issue here. I recall the type system being unhappy until I provided fragments that had voting system. What was the issue you were encountering @Will-Howard ?